cli/cmd/plugin/ui/web/tanzu-ui/.eslintrc.json

```diff
@@ -0,0 +1,66 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "react-app",
+        "react-app/jest",
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:prettier/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint",
+        "prettier"
+    ],
+    "rules": {
+        "indent": [
+            "warn",
+            4,
+            { "SwitchCase": 1 }
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "max-len": [
+            "warn",
+            {
+            "code": 140,
+            "comments": 120,
+            "ignoreUrls": true
+            }
+        ],
+        "@typescript-eslint/no-explicit-any": "off",
+        "no-duplicate-imports": "warn",
+        "no-multiple-empty-lines": "error",
+        "no-unused-vars": "warn",
+        "object-curly-spacing": [
+            "warn",
+            "always"
+        ],
+        "operator-linebreak": [
+            "error",
+            "after"
+        ],
+        "quotes": [
+            "warn",
+            "single"
+        ],
+        "semi": [
```

Comment:
I'd like to remove the "semi" as an error

Reply:
Agreed it doesn't need to be an error, but I would like to see it as a warning